src/NuGet.Core/NuGet.Configuration/Settings/SettingValue.cs

```diff
@@ -86,5 +115,10 @@ public override int GetHashCode()
         {
             return Tuple.Create(Key, Value, IsMachineWide).GetHashCode();
         }
+
+       /// <summary>
+       /// relative path source value in NuGet.Config
+       /// </summary>
+        public string OriginalValue { get; set; }
```

Comment:
1. Can you move the new property next to 'Value' property?
2. I would change the summary to be 'original value of the source as in NuGet.Config' instead of saying it is the relative path.